packages/rocketchat-authorization/lib/rocketchat.js

```diff
@@ -1 +1,5 @@
 RocketChat.authz = {};
+
+export const permissionLevel = {
+	SETTING: 'setting'
```

Comment:
I can imagine that there are other levels as well, such as the permission to change dedicated roles, thus making it a map